packages/react/src/components/Video/Video.tsx

```diff
@@ -98,6 +98,6 @@ class Video extends UIComponent<WithAsProp<VideoProps>> {
 Video.create = createShorthandFactory({ Component: Video, mappedProp: 'src' })
 
 /**
- * An video is a graphicical and audio representation of something.
+ * A Video provides ability to embed video content.
```

Comment:
And how is it different from Embed?

Reply:
`Embed`: An Embed displays content from external websites, like Instagram post or Youtube video.

Should we somehow emphasize more on the `external` part there? Or, maybe, replace `Youtube video` with `Google Map`?